### DISCUSSION TITLE
Unable to get ISR working in production

### DISCUSSION THREAD

**Discussion by spencersmb** [vercel/next.js#29979] [Help]
Put up a simple site here pulling data from my wordpress CMS using Apollo following the with-apollo setup.
https://et-headless-wp.vercel.app/

Everything works properly when in dev mode, as well as running npm build and npm start - revalidate seems to grab the latest posts and it all updates the site. However when I push to production it apollo will not update the cache like it does locally. Below is my getStaticProps and my apollo client file is identical to the with-apollo example for next.js...
Thanks!

```
export async function getStaticProps(){
  console.log('revalidate')

  const apolloClient = initializeApollo()

  await apolloClient.query({
    query: QUERY_ALL_POSTS,
  })

  await apolloClient.query({
    query: QUERY_POST_PER_PAGE,
  });

  return addApolloState(apolloClient, {
    props: {},
    revalidate: 1,
  })
}
```

**Comment by alexandrosk** [comment]
Same problem for me, please let me know if you have any update. 
I'm also using apollo, and I can't see the page as ISR on the 'next build' command
